src/opera/pge/base/base_pge.py

```diff
@@ -681,9 +684,15 @@ def augment_measured_parameters(self, measured_parameters):
             data_type = descriptions[name].get('attribute_data_type', guessed_data_type)
             attr_type = descriptions[name].get('attribute_type', "!Not Found!")
             attr_name = descriptions[name].get('display_name', guessed_attr_name)
+            escape_html = descriptions[name].get('escape_html', False)
 
-            augmented_parameters[name] = (dict(name=attr_name, value=value, attr_type=attr_type,
-                                               attr_description=attr_description, data_type=data_type))
+            if escape_html:
+                value = html.escape(value)
```

Comment:
If I understand correctly, this is to ensure that the generated XML isn't broken by special characters &c in the metadata. If that's the case, should we just escape/sanitize the value by default?

Reply:
I would prefer to take an explicit approach with escaping for only those fields that we're certain will contain a URL. I worry about potential unintended side-effects by applying it to every parameter.